doc/usecases/record_elektra/UC_interactive_configuration.md

```diff
@@ -0,0 +1,25 @@
+# Use Case: Interactively creating system config
+
+## Summary
+
+- **Title:** Interactively creating system config
+- **Scope:** Configuration
+- **Level:** User Goal
+- **Actors:** User (usually sysadmin)
+- **Brief:** In order to interactively create a workable system configuration, sometimes multiple iterations are required. 
+It should be possible to modify the configuration on a single host, and then apply the changes onto all other hosts.
+To integrate seamlessly into an Ansible workflow, it should also be possible to merge the changes into the original playbook.
+
+## Scenarios
+
+- **Precondition:** -
+- **Main success scenario:** 
+  - User connects to the host and changes settings in Elektra
+  - User exports a playbook that contains either
+    - only the changed keys
+    - the original playbook with the current changes applied
+- **Alternative scenario:** If the host was not configured using ansible-elektra, it is only possible to export a playbook with the changed keys
```

Comment:
```suggestion
- **Alternative scenario:** If the host has no applied playbook, it is only possible to export a playbook with the changed keys
```

Reply:
Hmm we can't do it with every playbook. It must be a playbook that was specifically exported by our plugin to do this, as we need to copy the playbook to the remote hosts for the merging to work.

Reply:
I did not intend to change the meaning in this way. You can simply ignore the suggestion.